pkgs/applications/science/physics/elmerfem/default.nix

```
@@ -0,0 +1,53 @@
+{ stdenv, fetchFromGitHub, cmake, git, gfortran, openmpi, blas, liblapack, qt4, qwt6_qt4, pkgconfig }:
+
+stdenv.mkDerivation rec {
+  pname = "elmerfem";
+  version = "8.4";
+
+  hardeningDisable = [ "format" ];
+
+  src = fetchFromGitHub {
+    owner = "elmercsc";
+    repo = "elmerfem";
+
+    rev = "release-${version}";
+    sha256 = "0vk31lplxlng173q8jjcpbyj1gaf98jvkqjvi9077d1nslya7vpm";
+    #rev = "b2a9134a9d3d883f733f5053e5db317b24211db2";
+    #sha256 = "0xgrpa1016rdq8qch8rhs91k9zrk3dh86rg14cz505x4ni5g4488";
+  };
+
```

src usually comes after version, hardening is usually after src
```suggestion
  src = fetchFromGitHub {
    owner = "elmercsc";
    repo = "elmerfem";

    rev = "release-${version}";
    sha256 = "0vk31lplxlng173q8jjcpbyj1gaf98jvkqjvi9077d1nslya7vpm";
    #rev = "b2a9134a9d3d883f733f5053e5db317b24211db2";
    #sha256 = "0xgrpa1016rdq8qch8rhs91k9zrk3dh86rg14cz505x4ni5g4488";
  };
  
  hardeningDisable = [ "format" ];
```